.github/workflows/ruby.yml

```diff
@@ -364,7 +366,8 @@ jobs:
       - name: Install vcpkg
         shell: bash
         run: |
-          ci/scripts/install_vcpkg.sh "${VCPKG_ROOT}"
+          # Install a more recent vcpkg than what's in .env, as we need recent fixes to the gobject-introspection port
+          ci/scripts/install_vcpkg.sh "${VCPKG_ROOT}" 2024.04.26 "$(pwd)/ci/vcpkg/ports-2024.04.26.patch"
```

Comment:
Should we just update the vcpkg version in `.env` instead? I wasn't sure what the impact of doing that might be.

Reply:
Yes.
Could you split this to a separated PR because wheel/jar builds also depend on vcpkg.

Reply:
I've opened #42167 for this, this has caused some build failures for the Python/Java builds which may take a little time to work through